mbr-format.opam

```
@@ -14,7 +14,7 @@ depends: [
   "ocaml" {>= "4.08.0"}
   "dune" {>= "3.4.0"}
   "lwt"
-  "cstruct" {>= "6.0.0"}
+  "cstruct" {>= "6.2.0"}
```

Since mbr_inspect.exe currently doesn't get installed when installing this opam package we don't need to raise the lower bound. We can specify bounds that are only valid with dev, e.g. `"cstruct" {dev & >= "6.2.0"}`, but this wouldn't work because then we don't have a constraint when it's not "dev" context.

---

okay, I think it's best to downgrade cstruct back to 6.0.0 to be sure it doesn't affect other projects that depend on the main MBR library. I'll think of a workaround to print the hex values

---

I think this might work:
```suggestion
  "cstruct" {>= "6.0.0"}
  "cstruct" {dev & >= "6.2.0"}
```